client/src/modules/login/signup.html

```diff
@@ -68,5 +68,7 @@
     <div class="login-other-container-item flex flex-center">
       <a ui-sref="login">Already have an account? Login here</a>
     </div>
+    <br/>
+    <div class="login-other-container-item flex flex-center"><a href="./auth/google" target="_self"><img src="images/btn_google_signin_light_normal_web@2x.png" alt=""></a></div>
```

Comment:
let's make this match the signin buttons for size, for now something just looks a bit off with the styling, i think it's just the sizing proportion... but we'll see